packages/cli-doctor/src/tools/versionRanges.ts

```diff
@@ -9,4 +9,5 @@ export default {
   ANDROID_NDK: '>= 19.x',
   // iOS
   XCODE: '>= 12.x',
+  COCOAPODS: '>= 1.9.1',
```

Comment:
let's verify the latest recommended version. I recall it was 1.10.x a few versions ago